openapi.yaml

```diff
@@ -3645,30 +3651,35 @@ components:
                     description: A bool representing whether or not to require MFA for requesting access to this group.
                     example: False
                     type: boolean
+                    x-speakeasy-ignore: true
```

Comment:
nixed because `require_mfa_to_request` should come from the request_configuration. top level field is deprecated